All Core Devs Meetings/Meeting 123.md

```diff
@@ -0,0 +1,303 @@
+
+**Tim Beiko** 
+* Yeah, that seems reasonable. So, okay. Let's move this to last call and, yeah. Keep track of the different clients who implement it and we'll move it to the final once. basically the, the four clients here have had implemented. 
+
+**Pooja | ECH**
+* So I would like to mention one thing he had when the proposal was originally proposed, it was under the networking type, now it has been moved from networking to core because it was believed that it's going to bring some of the consensus changes for, clique part. I'm wondering if that would be following the usual core EIP process for getting maybe a part of the upcoming upgrade. 
+
+**Tim Beiko** 
+* Well, because it's a clique it's not going to be. 
+
+**Pooja | ECH**
+* Yeah, on the mainnet. 
+
+**Danno Ferrin**
+* And it's a very compatible, it's not a hard fork. It'd be more equivalent to a soft fork because you can come onto a network and not have this implemented. And for the most part still participate. So it's, you're not going to be too surprised. You might take a while for you to figure out what the real light chain is, but eventually you'll figure it out. 
+
+**Pooja | ECH**
+* Sounds good. 
+
+**Tim Beiko** 
+* Okay. yeah, there's a question about, like, not sure if this should be a core EIP, Daniel, do you know off the top of your head, if the clique EIP itself is Core EIP? 
+
+**Daniel**
+* I know. I don't want to look it up. 
+
+**Tim Beiko** 
+* Okay. but I guess regardless of the actual stat, category, I think it makes sense to move it to fi to last call now. and then move it. The final ones clients have implemented it. Does anyone object to that? Okay. Yeah. 
+
+**Danno Ferrin**
+* Yeah. 225 is the click EIP and that is core. 
+
+**Tim Beiko** 
+* We can discuss this offline if people feel very strongly about it, but it seems like intuitively the extension, the clique should also be core if the actual clique EIP is core. but I'm not willing to die on that hell. And if people are that we should just discuss this on the discord. 
+
+**Pooja | ECH**
+* I just to get some more context, it was discussed in the last EIP meeting. So maybe that recording of that might help. we were discussing about networking proposals with Felix, Micah, and then generally it was decided that it should not be a part of networking EIP and should be moved to core EIP. So this is a recent change and we can follow the discussion there. 
+
+## EIP 2976 updates [29.08](https://youtu.be/-8TSQCwITA0?t=1738)
+
+**Tim Beiko** 
+* Okay. thank you, Daniel. We have another one. Micah is not on the call and I think this was his, but it was around the EIP 2976, the type transactions over gossip. I dunno if anyone else had context here, otherwise we can, we can talk about it another time. 
```

Comment:
```suggestion
* Okay. thank you, Danno. We have another one. Micah is not on the call and I think this was his, but it was around the EIP 2976, the type transactions over gossip. I dunno if anyone else had context here, otherwise we can, we can talk about it another time. 
```